Manifests/ManifestsApple/com.apple.mcxprinting.plist

```diff
@@ -0,0 +1,441 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+	<dict>
+		<key>pfm_description</key>
+		<string>Printing settings</string>
+		<key>pfm_description_reference</key>
+		<string>The payload for configuring printers. Removing this profile from a device does not automatically remove printers from the device.</string>
+		<key>pfm_documentation_url</key>
+		<string>https://developer.apple.com/documentation/devicemanagement/printing</string>
+		<key>pfm_domain</key>
+		<string>com.apple.mcxprinting</string>
+		<key>pfm_format_version</key>
+		<integer>1</integer>
+		<key>pfm_interaction</key>
+		<string>exlusive</string>
+		<key>pfm_ios_min</key>
+		<string>7.0</string>
+		<key>pfm_last_modified</key>
+		<date>2022-02-12T18:58:44Z</date>
+		<key>pfm_macos_min</key>
+		<string>10.7</string>
+		<key>pfm_platforms</key>
+		<array>
+			<string>macOS</string>
+		</array>
+		<key>pfm_subkeys</key>
+		<array>
+			<dict>
+				<key>pfm_default</key>
+				<string>Configures AirPrint settings</string>
```

Comment:
Is reference to AirPrint just leftover from originally cloned manifest?

Reply:
Ugh, yep, that is exactly what I used as my base to build this one.